docs/pages/guide/[slug].tsx

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+import matter from 'gray-matter';
+import { marked } from 'marked';
+import { useRouter } from 'next/router';
+import path from 'path';
+
+interface Props {
+  slug: string;
+  frontmatter: { title: string; href: string; description: string };
+  content: string;
+}
+
+const PackagePage: React.FC<Props> = ({ frontmatter: { title }, content }) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <div style={{ color: 'blue' }}>
+        <div onClick={() => router.back()}>&larr; Go Back</div>
```

Comment:
Tiffany would know for sure, but I don't think divs with `onClick` handlers are very accessible. I'd prefer a link to the main guides page here.

Reply:
A link would be preferable, but if you do want to keep this as a div, it will need the following attributes added to it: 
`role="button"`
`tabIndex={0}`
`aria-label="Go Back"`

The aria label should be added regardless of whether this is a div or a Link, though, since the arrow character could cause confusion when interpreted by a screen reader.